package.json

```diff
@@ -28,33 +28,33 @@
   ],
   "author": "Vojta Jina <vojta.jina@gmail.com>",
   "dependencies": {
-    "jasmine-core": "^3.6.0"
+    "jasmine-core": "^4.0.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "^12.1.4",
-    "@commitlint/config-angular": "^12.1.4",
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.1",
-    "@semantic-release/github": "^7.2.3",
-    "@semantic-release/npm": "^7.1.3",
-    "eslint": "^8.11.0",
+    "@commitlint/cli": "^16.2.3",
+    "@commitlint/config-angular": "^16.2.3",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^8.0.4",
+    "@semantic-release/npm": "^9.0.1",
+    "eslint": "^8.12.0",
     "eslint-config-standard": "^14.1.1",
```

Comment:
Didn't update this to the latest version due to the `no-var` warnings. I'll leave this to someone else.

Reply:
Can you elaborate on this issue a bit?

> BTW how am I supposed to construct the commit message so that linting is passing?

We are using [Angular's Commit Message Format](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit), which for breaking changes instead of using the exclamation point after the commit type, uses a special kind of footer.

Reply:
The newer eslint-config-standard versions have `no-var` set to `warn` so we are flooded with warnings. I suggest someone else deals with this later if/when the supported browsers change.

Reply:
Please excuse me but I still fail to understand. Let's get this conversation resolved to unblock this PR and continue the discussion in https://github.com/karma-runner/karma-jasmine/issues/315.